README.md

```diff
@@ -6,3 +6,45 @@
 
 
 [![Coverage](https://sonar.ownerofglory.com/api/project_badges/measure?project=Hillel-Trip-Planner&metric=coverage&token=sqb_886143ce12e2f96bb54f38ad77e4dc0b15d6fd22)](https://sonar.ownerofglory.com/dashboard?id=Hillel-Trip-Planner)
+
+
+## Prerequisites
+
+- Java Development Kit (JDK) version 17.
+- Apache Maven version 4.0.
+
+## Tools Used
+
+- Spring Framework
+- Hibernate ORM
+- Apache Tomcat (for deploying the web application)
+
+## Local Development
+
+### Testing
+
+To run tests, execute the following command:
+
+```bash
+./mvnw --batch-mode test
+```
+
+### Running
+
+Environment variables
+
+    db.driver=${MYSQL_DRIVER}
```

Comment:
env variables are usually written in this way for example:

MYSQL_DRIVER=com.mysql.cj.jdbc.SDriver

and so on